src/Hosting.CommandLine/Internal/CommandLineLifetime.cs

```diff
@@ -28,7 +28,7 @@ internal class CommandLineLifetime : IHostLifetime, IDisposable
         public CommandLineLifetime(IApplicationLifetime applicationLifetime,
             ICommandLineService cliService,
             IConsole console,
-            IUnhandledExceptionHandler unhandledExceptionHandler)
+            IUnhandledExceptionHandler unhandledExceptionHandler = null)
```

Comment:
Why is this necessary? This service should always be defined, right?

https://github.com/natemcmaster/CommandLineUtils/blob/1bdec5a956b93bcbd8433034a81439eb6fd9b0b7/src/Hosting.CommandLine/HostBuilderExtensions.cs#L42-L43

Reply:
It currently is. Ultimately I hoped that #196 would render the handler optional and I then would have removed the quoted line. Also I wanted to know if you can have optionally injected dependencies and it is consistent with the call in https://github.com/natemcmaster/CommandLineUtils/blob/1bdec5a956b93bcbd8433034a81439eb6fd9b0b7/src/Hosting.CommandLine/Internal/CommandLineLifetime.cs#L69